docs/configuration/trap-configuration.md

```diff
@@ -105,16 +105,16 @@ traps:
     nodePort: 30000
 ```
 
-Using this method, SNMP trap will always be forwarded to one of the trap receiver pods listening on port 30000 (as in the
-example above, remember - you can configure any other port). So doesn't matter IP address of which node you use, adding
-nodePort will make it end up in a correct place everytime. 
+Using this method, the SNMP trap will always be forwarded to one of the trap receiver pods listening on port 30000 (like in the
+example above, you can configure to any other port). So, it doesn't matter that IP address of which node you use. Adding
+nodePort will make it end up in the correct place everytime. 
 
-Here, good practice is to create IP floating address/Anycast pointing to the healthy nodes, so the traffic is forwarded in case of the
-failover. The best way is to create external LoadBalancer which balance the traffic between nodes.
+A good practice is to create an IP floating address/Anycast pointing to the healthy nodes, so the traffic is forwarded in case of the
+failover. To do this, create an external LoadBalancer that balances the traffic between nodes.
 
 ### Define number of traps server replica
-`replicaCount` defines that the number of replicas for trap container should be 2x number of nodes. The default value is `2`. 
-Example:
+`replicaCount` defines that the number of replicas for each trap container should be 2 times the number of nodes. The default value is `2`. 
```

Comment:
I think word each is not necessary here.

Reply:
Perhaps you guys are trying to say "`replicaCount` defines that the number of replicas per trap container should be 2 times the number of nodes"?

Reply:
Yes, that would be correct